components/omnibox/browser/brave_bookmark_provider.cc

```diff
@@ -41,13 +39,12 @@ void BraveBookmarkProvider::Start(const AutocompleteInput& input,
       continue;
     }
 
-    // We only allow the bookmark to be the default match if the input is
-    // literally contained in the title or URL.
+    // We only allow the bookmark to be the default match if the input is an
+    // exact match for the bookmark title
     auto lower_description = base::ToLowerASCII(match.description);
-    auto bump_match =
-        base::Contains(base::ToLowerASCII(match.contents), lower_text) ||
-        base::Contains(lower_description, lower_text);
-    if (!bump_match) {
+    auto description_matches =
+        lower_description == base::ToLowerASCII(input.text());
+    if (!description_matches) {
```

Comment:
nit: can be more intuitive w/o additional logical negative operation.
```suggestion
    if (lower_description != base::ToLowerASCII(input.text()) {
```

Reply:
much nicer!